docs/pages/admin-guides/management/operations/breakglass-access.mdx

```diff
@@ -0,0 +1,218 @@
+
+Log into the remote OpenSSH server and run the following commands.
+
+1. Create the `breakglass` user:
+
+```bash
+sudo adduser breakglass
+```
+
+Set a password for `breakglass` when prompted.
+
+2. Open the sudoers File for Editing:
+
+```bash
+sudo visudo
+```
+
+3. Grant limited sudo permissions:
+
+Add the following lines to restrict user switching and allow the `breakglass` user only to specific priviledged commands (e.g.,`cat`, `ls`, `journalctl`, `systemctl`, and `vim`):
+
+```bash
+breakglass ALL=(ALL) NOPASSWD: !ALL
+breakglass ALL=(ALL) NOPASSWD: /usr/bin/cat, /usr/bin/ls, /usr/bin/journalctl, /usr/bin/systemctl, /usr/bin/systemcl
+breakglass ALL=(ALL) NOPASSWD: !/usr/bin/su, !/usr/bin/sudo
+```
+
+`ALL`: Allows the rule to apply on all hosts (for local systems, this will be fine).  
+`(ALL)`: Allows the user to run the commands as any user.  
+`NOPASSWD: !ALL`: Prevents the user from running any command as root without a password prompt.  
+`NOPASSWD: /usr/bin/cat, /usr/bin/ls, /usr/bin/journalctl, /usr/bin/systemctl, /usr/bin/vim`: Allows the user to run only the specified commands without a password prompt.  
+`NOPASSWD: !/usr/bin/su, !/usr/bin/sudo`: Prevents the user from using `su` or `sudo` to switch users or run other commands with elevated privileges.  
+
+**Note:** Depending on the system you're working with, the path for each utility may be different. Confirm the path for each command you wish to grant priviledged access to before adding it to the sudoers file. 
+
+4. Save and Exit visudo:
+
+If you're using `nano` (default) as your sudo editor, press `Ctrl+X`, then `Y`, and press `Enter`.
+
+5. Restrict ability to switch users for all users not part of the `sudo` (or equivalent) group: 
+
+`su` uses [PAM](../../../../pages/enroll-resources/server-access/guides/ssh-pam.mdx#background) for authentication. In order to prevent the `breakglass` user from using `su` and backdooring access to other users, we can run the following to ensure that only users who are members of the `sudo` group can run `su`.
+
+```bash
+echo "auth required pam_sudo.so use_uid" > /etc/pam.d/su
+```
+
+## Step 3/6. Restrict cert-based SSH access to the `breakglass` user only
+
+The `/etc/ssh/authorized_principals` directory is used in conjunction with OpenSSH's certificate-based authentication mechanism to configure fine-grained control over which users or roles are allowed to authenticate using SSH certificates.
+We will use this mechanism to restrict cert-based SSH access only to the `breakglass` user to prevent other Teleport users from being able to generate client certificates and gain access to the server outside of Teleport.
+
+1. Ensure the following line is added and/or uncommented in your `/etc/ssh/sshd_config` file:
+
+```
+AuthorizedPrincipalsFile /etc/ssh/authorized_principals/%u
+```
+
+2. Create an authorized_principals file for the `breakglass` user 
+
+```bash
+echo "breakglass" | sudo tee /etc/ssh/authorized_principals/breakglass
```

Comment:
Instead of this whole machinery, can't we just create a `breakglass` user and make it trust the appropriate CA by putting it in `~breakglass/.ssh/authorized_keys` with the appropriate principals restriction?

Reply:
We could even do that for the `root` user, and make it require a principal of `root-breakglass` or something that won't accidentally be assigned as a login name to regular Teleport users.

Reply:
> Instead of this whole machinery, can't we just create a breakglass user and make it trust the appropriate CA by putting it in ~breakglass/.ssh/authorized_keys with the appropriate principals restriction?

I actually had this in the original setup but after some discussion with a few other folks went with the `authorized_principals` approach instead for the PR. Happy to switch it back if you think it simplifies things, and I like your recommendation of also doing the same thing for the `root` user so if makes sense to switch back I can tackle both at once in the updated commit.